.github/workflows/submitExtension.yml

```diff
@@ -0,0 +1,72 @@
+name: Submit extension to browser store
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Bump Version
+        default: v1.0.0
+        required: true
```

Comment:
this Action will present the user with a form field in the github UI to enter a string for version number